step.yml

```diff
@@ -1,7 +1,39 @@
 title: Manage iOS Code Signing
-summary: TODO
+summary: Automatically manages your code signing files.
 description: |-
-  TODO
+  The **Manage iOS Code Signing** Step takes care of automatically code signing your project before it's built on Bitrise.
+  The Step:
```

Comment:
@godrei shouldn't we add to the list that this step downloads signing certificates from Bitrise? I think it's a really important detail